ENIGMAsystem/SHELL/Platforms/xlib/XLIBwindow.cpp

```diff
@@ -45,8 +45,18 @@ Cursor NoCursor, DefCursor;
 
 using namespace enigma::x11;
 
+namespace tmpSize {
+    
+int tmpW = enigma::windowWidth;
+int tmpH = enigma::windowHeight;
+
+} // namespace tmpSize
+
 namespace enigma {
 
+unsigned long decorationsPrevious = !showBorder;
+unsigned long decorationsCurrent = showBorder;
```

Comment:
If these variables are not used in another source, they go to namespace anonymous. The `enigma::` namespace is only for internal variables used across compilation units.

```cpp
namespace {

} // anonymous namespace
```